File: Java+/BigData+/Databricks+/DatabricksJavaSdk/test/databricks/workspace/ClustersIT.java
package databricks.workspace;

import com.databricks.sdk.WorkspaceClient;
import com.databricks.sdk.mixin.ClustersExt;
import com.databricks.sdk.service.compute.ListClustersRequest;
import org.junit.jupiter.api.Test;

class ClustersIT {
    private final WorkspaceClient w = new WorkspaceClient();
    private final ClustersExt clusters = w.clusters();

    @Test
    void listClusters() {
        var request = new ListClustersRequest();
        for (var cluster : clusters.list(request)) {
            System.out.println(cluster.getClusterName());
        }
    }
}

File: Scala+/ScalaSbt213/Libs+/Databricks+/DatabricksJavaSdk/test/databricks/ClustersIT.scala
package databricks

import com.databricks.sdk.WorkspaceClient
import com.databricks.sdk.mixin.ClustersExt
import com.databricks.sdk.service.compute.ListClustersRequest
import org.scalatest.flatspec.AnyFlatSpec

class ClustersIT extends AnyFlatSpec {
  private val w = new WorkspaceClient
  private val clusters: ClustersExt = w.clusters

  it should "list clusters" in {
    val request = new ListClustersRequest
    val clusterList = clusters.list(request)
    clusterList.forEach(println)
  }
}